server/src/domain/usecases/createTask.ts

```diff
@@ -0,0 +1,15 @@
+import { Task, TaskProps } from '../entities/task'
```

Comment:
I guess this name is a bad name. CreateTask implies that there is logic and action. However, a domain is just data store, no logic.

Reply:
I relocated use-cases in application layer, and renamed the createTask.ts to createTaskUseCase.ts